traversal-tests/src/test/scala/overflowdb/traversal/TraversalTests.scala

```diff
@@ -122,55 +125,56 @@ class TraversalTests extends AnyWordSpec with ExampleGraphSetup {
     val artist1 = graph.addNode(Artist.Label)
     val artist2 = graph.addNode(Artist.Label)
 
-    val traversal: Traversal[Artist] = Traversal(graph.nodes).collectAll[Artist]
+    val traversal: Iterator[Artist] = graph.V().asScala.collectAll[Artist]
     traversal.l shouldBe Seq(artist1, artist2)
   }
 
   ".union step allows to aggregate multiple steps into one traversal" in {
-    def unionTrav = Traversal.fromSingle(center).out.union(_.out, _.in)
+    def unionTrav = Iterator.single(center).out.union(_.out, _.in)
     unionTrav.toSet shouldBe Set(l2, center, r2)
     unionTrav.l.sortBy(_.property(Thing.Properties.Name)) shouldBe List(center, center, l2, r2)
 
     // ensure that types are being preserved...
     val verifyIsNodeTraversal: Traversal[Node] = unionTrav
-    val verifyIsThingTraversal: Traversal[Thing] = Traversal.fromSingle(center).union(_.followedBy, _.followedBy)
+    val verifyIsThingTraversal: Traversal[Thing] = Iterator.single(center).union(_.followedBy, _.followedBy)
   }
-
+  /*
```

Comment:
what's the plan with these commented-out tests?

Reply:
As you suspected, they can go away and were left over from refactoring.